subprojects/core/src/test/groovy/org/gradle/initialization/buildsrc/BuildSrcUpdateFactoryTest.groovy

```diff
@@ -41,7 +41,7 @@ class BuildSrcUpdateFactoryTest extends Specification {
 
         then:
         classpath.asFiles == [new File("dummy")]
-        1 * listenerFactory.create(_) >> listener
+        1 * listenerFactory.create() >> listener
     }
 
     def "uses listener with rebuild off when marker file present"() {
```

Comment:
Should not have "when rebuild off" in the name"

Reply:
I removed both tests since they do not provide any additional value.